src/state/apis/nft/utils.ts

```diff
@@ -0,0 +1,53 @@
+import type { AccountId, ChainId } from '@shapeshiftoss/caip'
+import {
+  avalancheChainId,
+  bscChainId,
+  ethChainId,
+  fromAccountId,
+  optimismChainId,
+  polygonChainId,
+} from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import { invert } from 'lodash'
+
+// addresses are repeated across EVM chains
+export const accountIdsToEvmAddresses = (accountIds: AccountId[]): string[] =>
+  Array.from(
+    new Set(
+      accountIds
+        .map(fromAccountId)
+        .filter(({ chainId }) => isEvmChainId(chainId))
+        .map(({ account }) => account),
+    ),
+  )
+
+export enum SupportedOpenseaNetwork {
+  Polygon = 'matic',
+  Optimism = 'optimism',
+  Ethereum = 'ethereum',
+  BinanceSmartChain = 'bsc',
+  Avalanche = 'avalanche',
+  // Klaytn = 'klaytn',
+  // Arbitrum = 'arbitrum',
+  // Solana = 'solana',
+}
+
+// Opensea network <-> ChainId, shared amongst NFT resolvers
+// Move me to openseaApi/validators.ts if/when we implement Opensea API
```

Comment:
Pretty much what the comment says - needed to make the link *akschually* correct, which somehow worked with Zapper/mainnet out of pure luck